TerriaJS/terriajs
# upgrade husky to v8

### What this PR does

husky in v7 changed the way it works internally and it should provide more performant, also the v4 version wasn't running always on my machine most of the times it just skipped it like it wasn't there

### Test me
  
- checkout the branch
- run yarn install
- try some dummy commit that should trigger prettier run and observe that pretty-quick fixed that file

### Checklist
